app/views/content/landing/careers/_header.html.erb

```diff
@@ -0,0 +1,6 @@
+<%= render Content::CampaignHeroComponent.new(
+  title: ["You're more of a teacher than", "you think"],
+  colour: "grow",
+  image: @front_matter["image"],
```

Comment:
I don't think we need to use an image from the ad here - probably better to use one of the brand campaign images

Reply:
@gemmadallmandfe do you mean switch the main hero image of Ben? Or the one underneath it? Or both? Thanks cc @jenhadfield-dfe

Reply:
@gemmadallmandfe Jen's changed the hero image if you want to check you're happy with it.

Reply:
Just meant switching the main hero image (as Jen has done). Looks good to me - but more the Pipeline team's call!